Index: vendor/symfony/doctrine-bridge/DependencyInjection/AbstractDoctrineExtension.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- vendor/symfony/doctrine-bridge/DependencyInjection/AbstractDoctrineExtension.php
+++ vendor/symfony/doctrine-bridge/DependencyInjection/AbstractDoctrineExtension.php
@@ -129,11 +129,15 @@
      */
     protected function setMappingDriverConfig(array $mappingConfig, $mappingName)
     {
-        if (!is_dir($mappingConfig['dir'])) {
+        $mappingDirectory = $mappingConfig['dir'];
+        if (!is_dir($mappingDirectory)) {
             throw new \InvalidArgumentException(sprintf('Invalid Doctrine mapping path given. Cannot load Doctrine mapping/bundle named "%s".', $mappingName));
         }
 
-        $this->drivers[$mappingConfig['type']][$mappingConfig['prefix']] = realpath($mappingConfig['dir']);
+        if (substr($mappingDirectory, 0, 7) !== 'phar://') {
+            $mappingDirectory = realpath($mappingDirectory);
+        }
+        $this->drivers[$mappingConfig['type']][$mappingConfig['prefix']] = $mappingDirectory;
     }
 
     /**
